bin/pretty-quick.js

```diff
@@ -14,63 +14,67 @@ const args = mri(process.argv.slice(2), {
   },
 });
 
-const prettyQuickResult = prettyQuick(
-  process.cwd(),
-  Object.assign({}, args, {
-    onFoundSinceRevision: (scm, revision) => {
-      console.log(
-        `🔍  Finding changed files since ${chalk.bold(
-          scm,
-        )} revision ${chalk.bold(revision)}.`,
-      );
-    },
+async function main() {
```

Comment:
Seeing as Prettier v3 is ESM only, pretty-quick can follow suite. That means adding `"type": "module` to the package.json and after that you can change this to have a top level await, thus removing the `async function main()` and just inlining the content.

It should then also be possible to remove the `babel` dependency, and adjust the other packaging related matters accordingly.

Reply:
This would be a breaking change though, so a major release is necessary for this

Depends on @azz to decide on that I guess

Reply:
prettier v3 is not ESM only. But it doesn't provide sync API.

Reply:
I'll merge this PR as-is first because it only affect `prettier` v3, but I'll upgrade all dependencies in another PR.